src/index.js

```diff
@@ -162,7 +164,7 @@ const sketch = function (p5) {
       tilePoints.push(p5.createVector().set(projectionCalculator3d.getProjectedPoint([i + 1, j + 1, height])))
       tilePoints.push(p5.createVector().set(projectionCalculator3d.getProjectedPoint([i + 1, j, height])))
 
-      currentStyle.drawTile(tilePoints, p5.createVector(i, j, height), height !== 0)
+      currentStyle.drawTile(tilePoints, p5.createVector(i, j, height), height !== 0, { i, j })
```

Comment:
you can't modify that part because this can affect other styles. Moreover it's useless, you can know i and j with the 3D vector given as 2sd argument. 
i == vect.x
j == vect.y 
:)